run_dir/design/worksets.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+
+<!-- 
+Template file: worksets.html
+URL: /worksets/[optional filter]
+Title: Summary of all Worksets
+Description: Shows a table with all worksets.
+-->
+
+{% block stuff %}
+
+{% if worksets == 'all' %}
+  <h1 id="{{ worksets }}" class="worksets_page_heading"><span id="page_title">All Worksets</span> <small>All worksets found in statusdb</small></h1>
+{% end %}
+
+
+<div id="loading_spinner" style="text-align:center; margin:100px 0;"><span class='glyphicon glyphicon-refresh glyphicon-spin'></span>  Loading worksets table..</div>
+<div id="page_content" style="display:none;">
+
+<form id="Search-form" class="form-inline marginBottom">
+  <div class="pull-right">
+    <div class="form-group" style="margin-right:15px;">
+      <div class="btn-group btn-group-sm" id="default_preset_buttons">
+        <button id="resetWorksetCols" data-action="filterReset" type="button" class="search-action btn btn-default">Default</button>
+      </div>
+    </div>
+    <!-- <button class="pull-right btn btn-primary" data-toggle="modal" data-target="#worksetFieldsModal">Filter Columns</button> -->
```

Comment:
I think you can comment out this entire form (`#Search-form`) can't you? Or do the other buttons do something?